hw02/EnemyCollision.cs

```diff
@@ -18,15 +18,23 @@ void Update()
     
     // TODO: Problem 7: Why did I not include the words public or private here?
     // What does it mean when I only write void (Movement)? What does void mean?
+    
+    //public means you can access it globally,while private you can not. if no modifier is provided, it defalts to private
+    //void is a return type, it means the function doesn't return a value.
```

Comment:
Yes. Also noting that not having a `public` or `private` in front means it is `private` by default. Therefore, `void Movement()` is a private class.